### ISSUE TITLE
DevWorkspace MVP (minimal viable product)

### ISSUE THREAD

**Issue by l0rd** [devfile/devworkspace-operator#357]
We would like to submit DevWorkspace as CNCF incubator project. Here we want to track tasks to accomplish before pushing the submit button:

**Mandatory**

- [x] [Projects clone should happen before other components are started](https://github.com/devfile/devworkspace-operator/issues/205)
- [x] Replace (or add) Apache-2.0 license
- [ ] Register a domain for the project landing page
- [ ] Setup the website content at https://github.com/devworkspace/devworkspace.github.io
- [x] Review and fix devfile v2 documentation
- [x] Remove Web Terminal Operator-specific code from DevWorkspace Operator and develop a plan to continue supporting WTO.

**Nice to have**

- [x] Publish DWO on the Operator Hub
- [x] Release devfile spec v2.1
- [ ] Record a demo of `<kubectl apply -f dw.yaml>` | `<kubectl exec>`  `<code (with vim)>` | `<build>` | `<run>`

**Comment by amisevsk** [comment]
Added a note about web-terminal-specific functionality that should probably be removed before going forward.

**Comment by amisevsk** [comment]
Closing as all CNCF-related stuff is done as far as I know (devfile/api is in CNCF, DWO will not be)

**Comment by l0rd** [comment]
For the records we decided to move forward with the Devfile rather than the DevWorskspace Operator to be a CNCF project. That's because there was a cross-team, cross-company interest in the Devfile but not in the DevWorkspace. At the time I created the [DevWorkspaec GitHub org](https://github.com/devworkspace) to host a DevWorkspace website but then we abandoned that plan. Now the DevWorkspace Operator is one subproject of the CNCF sandbox project Devfile.
